src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger.java

```diff
@@ -1346,6 +1347,27 @@ public void setTriggerOnEvents(List<PluginGerritEvent> triggerOnEvents) {
         this.triggerOnEvents = triggerOnEvents;
     }
 
+    /**
+     * Returns an iterator over the all gerrit projects configured for the trigger.
+     *
+     * @return an iterator over the all gerrit projects configured for the trigger.
+     */
+    private Iterator<GerritProject> getAllGerritProjectsIterator() {
+        if (gerritProjects != null && dynamicGerritProjects != null) {
+            return Iterators.concat(gerritProjects.iterator(), dynamicGerritProjects.iterator());
```

Comment:
I am normally opposed to using guava, but in this case it's using something existing from Jenkins core so it should be fine.